pkgs/by-name/go/gopass-jsonapi/package.nix

```diff
@@ -44,6 +45,47 @@ buildGoModule rec {
   postFixup = ''
     wrapProgram $out/bin/gopass-jsonapi \
       --prefix PATH : "${gopass.wrapperPath}"
+
+    # Generate native messaging manifests for Chrome and Firefox.
+    # It would be nicer to not do that in the fixupPhase but `gopass-jsonapi configure`
+    # is run which relies on on the wrapped `gopass-jsonapi`.
+    export HOME=$(mktemp -d)
+    ${gnupg}/bin/gpg --batch --passphrase "" --quick-generate-key "user <user@localhost>"
+    ${gopass}/bin/gopass setup --name "user" --email "user@localhost"
+
+    # The options after `--print=false` are of no effect, but if missing
+    # `gopass-jsonapi configure` will ask for them. (`--global` is used to compute
+    # the manifest path, but it is overridden by `--manifest-path` and `--libpath`
+    # seems to only be used to compute Firefox' global manifest path, see
+    # https://github.com/gopasspw/gopass-jsonapi/blob/v1.15.14/setup_others.go#L33-L46)
+    #
+    # Manifest paths were taken from
+    # https://github.com/gopasspw/gopass-jsonapi/blob/v1.15.14/internal/jsonapi/manifest/manifest_path_linux.go
+    #
+    # Both calls will write `$out/lib/gopass_wrapper.sh` but they are identical.
+    # https://github.com/gopasspw/gopass-jsonapi/blob/v1.15.14/internal/jsonapi/manifest/wrapper.go#L14-L36
+    #
+    # `gopass-jsonapi configure` ask for confirmation before writing any files,
+    # `echo y` gives it.
+    echo y | "$out"/bin/gopass-jsonapi configure \
+      --browser chrome \
+      --path "$out"/lib \
+      --manifest-path "$out"/etc/chromium/native-messaging-hosts/com.justwatch.gopass.json \
+      --print=false \
+      --global \
+      --libpath "$out"/lib
+    echo y | "$out"/bin/gopass-jsonapi configure \
+      --browser firefox \
+      --path "$out"/lib \
+      --manifest-path "$out"/lib/mozilla/native-messaging-hosts/com.justwatch.gopass.json \
+      --print=false \
+      --global \
+      --libpath "$out"/lib
```

Comment:
I think this is quite clean and we do not need a custom option `programs.firefox.nativeMessagingHosts.gopass`:

```nix
{
  programs.firefox = {
    nativeMessagingHosts.packages = [ pkgs.gopass-jsonapi ];
  };
}
```

How about we add a custom `gopass_wrapper.sh` to nixpkgs and just `cp ./gopass_wrapper.sh "$out"/lib/gopass_wrapper.sh` it below?